templates/enmasse-operator/050-Deployment-enmasse-operator.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         image: ${CONTROLLER_MANAGER_IMAGE}
         imagePullPolicy: ${IMAGE_PULL_POLICY}
         env:
+        - name: LOG_LEVEL
```

Comment:
I think the default should stay "info".

Reply:
I was thinking to leave it at debug for a few days in case there are more issues popping up in CI, and then move it back to info once it is stable.

Reply:
… and then to forget about it :grin: